mtf/metadata/setup.py

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env python
```

Comment:
sure, that is exactly this reason, it is independent project in MTF project, we can add it to masteter python and let there this file, or maybe call this child setup.py in case it is possible to do.